.github/workflows/release_dev_chart.yaml

```diff
@@ -0,0 +1,41 @@
+name: Release Charts
+
+on:
+  workflow_run:
+    workflows: ["unit tests"]
+    types:
+      - completed
+    branches:
+      - dev
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.9.2
+
+      - name: Seed helm deps
+        run: |
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.3.0
+        with:
+          mark_as_latest: false
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_SKIP_EXISTING: "true"
+          CR_RELEASE_NAME_TEMPLATE: "{{ .Name }}-{{ .Version }}-dev"
```

Comment:
is this the only line of difference?  if so we can probably template out the "-dev".

Reply:
this line and `mark_as_latest: false` are the only differences

Reply:
I did something that might be relevant [here](https://github.com/broadinstitute/seqr/blob/master/.github/workflows/hail-search-persistent-volume-snapshot-release.yaml#L65)